samples/SkiaAndroidSample/src/main/java/org/jetbrains/skiko/sample/MainActivity.kt

```diff
@@ -19,7 +21,7 @@ class MainActivity : Activity() {
             //layoutParams = ViewGroup.LayoutParams(1000, 1200)
         }
 
-        val skiaLayer = SkiaLayer()
+        val skiaLayer = SkiaLayer(SkikoGestureEventKind.values())
```

Comment:
Hmm, not sure if like this change. Skia layer shalln't be inited this way.

Reply:
Maybe smth like `layer.eventsToListen = arrayOf(...)`?

Reply:
Maybe it's a good idea indeed, I'm developing it now and also will try this approach.